### PR TITLE
Update gsibec submodule to tag 1.1.3

### LINKED ISSUE #854: Update to gsibec tag 1.1.3
GDASApp `develop` uses gsibec tag 1.1.2.   SABER requires gsibec tag 1.1.3.   This issue is opened to document the updating of the gsibec submodule to tag 1.1.3.

### PR DESCRIPTION
**Pull request by RussTreadon-NOAA**
JEDI library `saber` requires `gsibec` tag 1.1.3.   The `gsibec` submodule currently points at tag 1.1.2.   This PR updates the `gsibec` submodule to point at tag 1.1.3.   This update is necessary if developers want to run `fv3jedi_var.x` with the GSI static B.

Resolves #854

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]

**Comment by RussTreadon-NOAA** [comment]
Thank you @CoryMartin-NOAA .  I'll merge before 1 pm EST unless @guillaumevernieres or you have other PRs which need to get into `develop` before this PR is merged.

**Pull request merged**
